src/Orchard.Cms.Web/Modules/Orchard.OpenId/Controllers/AccessController.cs

```diff
@@ -515,9 +515,9 @@ public IActionResult Deny()
                     OpenIdConnectConstants.Scopes.OfflineAccess,
                     OpenIddictConstants.Scopes.Roles
                 }.Intersect(request.GetScopes()));
-            }
 
-            ticket.SetResources(request.GetResources());
+                ticket.SetResources(request.GetResources());
```

Comment:
@jersiovic note: in the future, we may want to add a "resources" setting to allow limiting the resources that can be used here (instead of just blindly storing the specified resources).